cg/store/models.py

```diff
@@ -64,8 +64,8 @@ class Customer(Model):
     project_account_ki = Column(types.String(32))
     project_account_kth = Column(types.String(32))
     organisation_number = Column(types.String(32))
-    invoice_address = Column(types.Text)
-    invoice_reference = Column(types.String(32))
+    invoice_address = Column(types.Text, nullable=False)
```

Comment:
Please write script to update database to reflect the model changes

Reply:
Emilia and Vallteri are taking care of adding missing data to those fields.

Reply:
Yes, I'm talking about the database structure, not the data

Reply:
I suggest adding a script, something like make_invoice_information_mandatory.sql, in the scripts folder doing something like: https://stackoverflow.com/questions/689746/altering-a-column-null-to-not-null